test/test-test-case.rb

```diff
@@ -65,14 +65,14 @@ def internal_backtrace_entry?(entry)
         entry.start_with?("<internal:")
       end
 
-      def normalize_location(location)
+      def normalize_backtrace(location)
         filtered_location = location.reject do |entry|
           jruby_backtrace_entry?(entry) or
             rubinius_backtrace_entry?(entry) or
             internal_backtrace_entry?(entry)
         end
         filtered_location.collect do |entry|
-          entry.sub(/:\d+:/, ":0:")
+          entry.sub(/:\d+:/, ":0:").sub(/in [`'](?:[^']+?[#.])?/, "in '")
```

Comment:
Can we simplify this?

```suggestion
          entry.sub(/:\d+: in [`']/, ":0:in '")
```

Reply:
I did this one too. Still, `(?:[^']+?[#.])?` is needed to remove the receiver from the backtrace.